### DISCUSSION TITLE
Update urllib3 dependency to 2.x?

### DISCUSSION THREAD

**Discussion by johanneskastl** [boto/botocore#2933] [General]
https://github.com/boto/botocore/blob/daf4110dc9be6f87fdaef0992499896c95997f91/setup.py#L28

Currently urllib3 only versions < 1.27 are supported, while urllib3 is already at 2.x. Could this be adapted to allow using this with newer versions?

**Comment by tim-finnigan** [comment]
Hi thanks for reaching out. This request is being tracked here: https://github.com/boto/botocore/issues/2926. There aren't plans to move the version pin in the immediate future due to urllib3 2.x introducing breaking changes in Lambda and AL2. Please refer to that issue for updates going forward.

**Reply by johanneskastl** [reply]
Thanks a lot for the reply, seem I have missed it in the list of issues. My apologies.
